specifyweb/workbench/upload/treerecord.py

```diff
@@ -243,6 +243,9 @@ def rescope_tree_from_row(self, row: Row) -> Tuple["ScopedTreeRecord", Optional[
         # Based on the target treedef, get the treedefitems and root for the tree    
         treedefitems = list(tree_rank_model.objects.filter(treedef_id=target_rank_treedef_id).order_by("rankid"))
         root = tree_node_model.objects.filter(definition_id=target_rank_treedef_id, parent=None).first()
+        if root is None:
+            logger.warning(f"No root found for treedef {target_rank_treedef_id}")
+            return self, WorkBenchParseFailure('noRoot', {}, None)
```

Comment:
I'll give you the benefit of doubt _again_, but workbench is supposed to handle no root cases (and it has done that since years ago you touched it). Now, after knowing this, don't you feel this is code smell? 

Don't you think that there could be a much bigger bug somewhere that we are not thinking about? Don't just patch things. I don't particularly care that we are behind timeline for this (what if the real bug causes a much bigger issue downstream, taking more testing resources + dev time?)

Just do things properly please, no hacks.